### PR TITLE
feat(fx-2702): updates fair header layout

### PR DESCRIPTION
**Pull request by dzucconi**
Closes: [FX-2702](https://artsyproduct.atlassian.net/browse/FX-2702)

localhost_5000_fair_1-54-paris-2021

This is according to the spec; a constant 600 height and full bleed width. I suspect that any recent fair will have to have its image updated since they used portrait crops. And I also suspect we'll want to tweak the mobile image to be shorter.

### CODE REVIEW

**Comment by dzucconi** [comment]
The other frustrating thing is how we handle logos:

They are uploaded with transparency: but Gemini doesn't support cropping with transparency (backgrounds go to black); so we can't crop the full sized image. So we have to use the sub par image and we can't resize it either to set it proportionally; so we're left with the floating whitespace.

File: src/v2/Apps/Fair/Components/FairHeader/FairHeader.tsx [hunk 1]
```diff
@@ -42,23 +51,20 @@ const FairHeader: React.FC<FairHeaderProps> = ({ fair, ...rest }) => {
...
-        gridColumnGap={3}
-        gridRowGap={2}
-        gridTemplateColumns={["repeat(1, 1fr)", `repeat(${columnCount}, 1fr)`]}
-        textAlign={columnCount === 1 ? "center" : undefined}
-      >
-        <Box>
-          <Text as="h1" variant="largeTitle">
-            {fair.name}
-          </Text>
+      <GridColumns mt={[2, 4]}>
```
Comment (anandaroop) [inline src/v2/Apps/Fair/Components/FairHeader/FairHeader.tsx:54]: Should we deprecate `CSSGrid` over in Palette?

File: src/v2/Apps/Fair/Components/FairHeader/FairHeaderIcon.tsx [hunk 2]
```diff
@@ -1,37 +1,28 @@
 import React from "react"
-import { Box, space } from "@artsy/palette"
+import { Box, BoxProps } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
 import { FairHeaderIcon_fair } from "v2/__generated__/FairHeaderIcon_fair.graphql"
 
-interface FairHeaderIconProps {
+interface FairHeaderIconProps extends BoxProps {
```
Comment (anandaroop) [inline src/v2/Apps/Fair/Components/FairHeader/FairHeaderIcon.tsx:6]: 🙏

File: src/v2/Apps/Fair/Components/FairHeader/FairHeaderImage.tsx [hunk 3]
```diff
@@ -1,91 +1,39 @@
...
-        </Container>
-      </Media>
-    </>
+    <Container bg="black10" height={600}>
+      <picture>
+        <source srcSet={image.lg.srcSet} media="(min-width: 1200px)" />
+        <source srcSet={image.md.srcSet} media="(min-width: 700px)" />
+        <source srcSet={image.sm.srcSet} media="(max-width: 700px)" />
+        <Image src={image.sm.src} alt="" loading="lazy" />
+      </picture>
```
Comment (anandaroop) [inline src/v2/Apps/Fair/Components/FairHeader/FairHeaderImage.tsx:35]: Ah nice, don't think I'd seen this in the wild before 😎

File: src/v2/Components/FullBleed.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+ * Breaks out of horizontal padding
+ */
+export const FullBleed = styled(Box)`
+  position: relative;
+  left: 50%;
+  right: 50%;
+  width: 100vw;
+  max-width: 100vw;
+  margin-left: -50vw;
+  margin-right: -50vw;
```
Comment (anandaroop) [inline src/v2/Components/FullBleed.tsx:14]: Clever!

**Review by anandaroop — APPROVED** [review]
Thanks for all the test refactoring too

File: src/v2/Apps/Fair/Components/FairHeader/FairHeader.tsx (hunk 1 above)
Comment (dzucconi) [reply src/v2/Apps/Fair/Components/FairHeader/FairHeader.tsx:54]: It's still useful as a generic kind of CSSGrid when u need to step outside of our 12 column layout. What we should deprecate is `Grid` though, which uses bootstrap grid under the hood.

**Pull request merged**